enos/enos.vars.hcl

```diff
@@ -130,7 +130,7 @@
 
 # vault_upgrade_initial_release is the Vault release to deploy before doing an in-place upgrade.
 # vault_upgrade_initial_release = {
```

Comment:
We should remove this from the example variables because it is no longer used and has instead been replaced by the matrix.